OPHD/UI/PopulationPanel.cpp

```diff
@@ -66,8 +66,8 @@ PopulationPanel::PopulationPanel(const Population* pop, const PopulationPool* po
 		imageCache.load("ui/skin/window_bottom_right.png")
 	}
 {
-	mPopulation = pop;
-	mPopulationPool = popPool;
+	mPopulation = &pop;
+	mPopulationPool = &popPool;
```

Comment:
Ahh, I see internally pointer syntax is still being used. For a moment I was wondering why this didn't result in a compiler error. If reference syntax had been used internally, then this initialization would have been too late. The constructor init list is needed to initialize references. Once the constructor body is entered, all component objects are already initialized.

Or was that why pointer syntax was still being used internally?